spec/ClearCheck.spec.js

```diff
@@ -32,4 +32,6 @@ describe('the ClearCheck.js class', () => {
 
 		expect(result).toBeDefined();
```

Comment:
why is that expected to be defined :P ?

Reply:
because the function returns a value. see src.